docs/developers/local-rules.md

```diff
@@ -185,6 +188,7 @@ semantic rules.
 +  )
 +lazy val `scalafix-tests` = (project in file("scalafix/tests"))
 +  .settings(
++    scalaVersion := _root_.scalafix.sbt.BuildInfo.scala2XX, //FIXME
```

Comment:
this is actually redundant, as the entire project must have a `scalaVersion` that fully matches the scalafix one, as documented in https://scalacenter.github.io/scalafix/docs/users/installation.html#sbt (which is referenced above).